src/CMakeLists.txt

```diff
@@ -787,7 +788,6 @@ else()
 			set(OTHER_FLAGS "${OTHER_FLAGS} -mfpmath=sse -msse")
 		endif()
 	endif()
-
```

Comment:
This should not be deleted (used as a separator).

Reply:
Added.